Formula/lmod.rb

```diff
@@ -0,0 +1,64 @@
+class Lmod < Formula
+  desc "Lua-based environment modules system to modify PATH variable"
+  homepage "https://www.tacc.utexas.edu/research-development/tacc-projects/lmod"
+  url "https://github.com/TACC/Lmod/archive/7.7.16.tar.gz"
+  sha256 "cf14183f9954bf0d4c22db4b7db3a6459c8cb607b9d1ff25e18e0989b48bf58b"
+
+  depends_on "lua"
+
+  resource "luafilesystem" do
+    url "https://github.com/keplerproject/luafilesystem/archive/v1_7_0_2.tar.gz"
+    sha256 "23b4883aeb4fb90b2d0f338659f33a631f9df7a7e67c54115775a77d4ac3cc59"
+  end
+
+  resource "luaposix" do
+    # Don't use v34.0.2, it's broken (at least on macOS)
+    url "https://github.com/luaposix/luaposix/archive/v34.0.1.tar.gz"
+    sha256 "43e7fb1fa4a8b905dd256703aa62116937be72d07d3a46b66a8e9b2264921129"
+  end
+
+  def install
+    luapath = libexec/"vendor"
+    ENV["LUA_PATH"] = "#{luapath}/share/lua/5.3/?.lua" \
+                      ";#{luapath}/share/lua/5.3/?/init.lua"
+    ENV["LUA_CPATH"] = "#{luapath}/lib/lua/5.3/?.so"
+
+    resources.each do |r|
+      r.stage do
+        system "luarocks", "make", "--tree=#{luapath}"
+      end
+    end
+
+    system "./configure",
+      "--disable-debug",
```

Comment:
First one goes on the same line as configure, then indent to align.
See `sile.rb` for an example.

Reply:
I expect I inherited this from the old formula. Much nicer now.

Reply:
Several of them weren't needed. Fixed.